packages/London-Boroughs/London-Boroughs.svg

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
```

Comment:
Could you fix the SVG following the [contribution guidelines](https://github.com/VictorCazanave/svg-maps/blob/master/CONTRIBUTING.md#svg-file) (no `<?xml version="1.0"?>`, no `width`/`height`...) please?

Reply:
The [SVG is still not valid](https://github.com/VictorCazanave/svg-maps/blob/master/CONTRIBUTING.md#svg-file):
- indentation is missing
- `id` of `<path>` tags cannot contain spaces and must be in kebab-case
- `name` attributes of `<path>` tags are missing